src/bundle/Resources/views/layout.html.twig

```diff
@@ -101,6 +101,15 @@
         %}
             <script type="text/javascript" src="{{ asset_url }}"></script>
         {% endjavascripts %}
+
+        {% javascripts
+            'bundles/bazingajstranslation/js/translator.min.js'
+            'js/translations/config.js'
```

Comment:
I think we should use `@EzPlatformAdminUiBundle`, and for `translator` we also should use bundle name.

Reply:
Indeed we should be able use `@BazingaJsTranslationBundle` in case of `translator.min.js`. I will fix this. 

This `config.js` file is generated by 
https://github.com/willdurand/BazingaJsTranslationBundle/blob/master/Resources/doc/index.md#the-dump-command in `/web/js/translations` folder (same as other file there).